store/tikv/2pc.go

```diff
@@ -1398,8 +1399,8 @@ func (c *twoPhaseCommitter) checkSchemaValid(ctx context.Context, checkTS uint64
 		err := errors.Errorf("mock check schema valid failure")
 		failpoint.Return(nil, false, err)
 	})
-	checker, ok := c.txn.us.GetOption(kv.SchemaChecker).(schemaLeaseChecker)
-	if !ok {
+	checker := c.txn.schemaLeaseChecker
+	if checker == nil {
```

Comment:
how about REMOVE var `checker` and `if c.txn.schemaLeaseChecker== nil `

Reply:
ok. Good idea.